client/src/components/modal.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styles from '../styles/modal.css';
+
+
+const Modal = (data) => {
+  const clickedIdx = 0;
+  return (
+    <div className={styles.modal}>
+      <div className={styles.modalContent}>
+        <div className={styles.modalImage}>
+          <img src={data.dataArr[6].photosAndComments[clickedIdx].imageUrl} alt="host home" />
```

Comment:
what does the [6] stand for in this case? Is it just hardcoding it to render Listing #6?

Reply:
Yes, currently hardcoding the index of the id that I will be receiving.